airflow/api_fastapi/execution_api/datamodels/asset.py

```diff
@@ -36,7 +36,7 @@ class AssetAliasResponse(BaseModel):
     group: str
 
 
-class AssetProfile(StrictBaseModel):
+class AssetProfile(BaseModel):
```

Comment:
Why is this change needed?

Reply:
This is not. It's something I accidentally changed I think 🤔 let me changed it back

Reply:
changed back